### PR TITLE
OSPOOL-131: Support ARM

### PR DESCRIPTION
**Pull request by mwestphall**
Refactor the build matrix to include platform as an argument. Add a second job to consolidate multi-platform artifacts under a single tag in the registry. See the following documentation for a reference implementation:

https://docs.docker.com/build/ci/github-actions/multi-platform/#distribute-build-across-multiple-runners

### CODE REVIEW

File: .github/actions/push-digest-local/action.yaml [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    #   github.event_name != 'pull_request' &&
+    #   github.repository_owner == 'opensciencegrid'
+    uses: docker/login-action@v2
+    with:
+      registry: ${{ inputs.registry }}
+      username: ${{ inputs.username }}
+      password: ${{ inputs.password }}
+
+  - id: upload-image
+    uses: opensciencegrid/build-container-action@420b64817b8a5d296a4201af6e955c4410107da5
```
Comment (mwestphall) [inline .github/actions/push-digest-local/action.yaml:61]:
```suggestion
    uses: opensciencegrid/build-container-action@HEAD
```

File: .github/actions/push-digest-local/action.yaml [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    with:
+      registry_url: ${{ inputs.registry }}
+      osg_series: ${{ inputs.osg_series }}
+      osg_repo: ${{ inputs.osg_repo }}
+      base_os: ${{ inputs.base_os }}
+      platform: ${{ inputs.platform }}
+      push_by_digest: true
+      timestamp: ${{ inputs.timestamp }}
+      setup: false
```
Comment (mwestphall) [inline .github/actions/push-digest-local/action.yaml:70]:
```suggestion
      buildx_setup: false
```

**Review by mwestphall — COMMENTED** [review]
Update build-container-action input args

**Comment by brianhlin** [comment]
Noting here that this required the already merged https://github.com/opensciencegrid/build-container-action/pull/12

**Review by brianhlin — APPROVED** [review]
LGTM but I've left some comments, mostly wanting more comments

**Comment by mwestphall** [comment]
Good call on the documentation, those should be updated now.

**Review by brianhlin — APPROVED** [review]
LGTM

**Pull request merged**